### ISSUE TITLE
Add update_manager to moonraker.conf for fluidd

### ISSUE THREAD

**Issue by noobydp** [th33xitus/kiauh#57]
Fluidd 1.2.0 added support for auto updates via the UI.
If you add the section below to the moonraker.conf file fluidd's update manager will start working.

https://github.com/cadriel/fluidd/releases/tag/v1.2.0

```
[update_manager]
client_repo: cadriel/fluidd
client_path: /home/pi/fluidd
```

**Comment by th33xitus** [comment]
I will not add a UI specific entry there since its not only Fluidd that supports the Moonraker update manager but Mainsail as well. So i rather think about giving the user a choice. I'll have a look.

**Comment by dewi-ny-je** [comment]
The syntax seems wrong. I think it should be

```
[update_manager client fluidd]
type: web
repo: cadriel/fluidd
path: ~/fluidd
```

**Comment by th33xitus** [comment]
@dewi-ny-je the syntax got updated a few days ago. at the time this issue was opened, syntax was fine :)

i hope i will be able to merge soon, some more tests need to be done, then the big kiauh rework is done.

**Comment by th33xitus** [comment]
Should be solved with 16e6dde99864bc0c315f483c036938728ae597ba

Closing...
